assets/js/src/views/toolbar.js

```diff
@@ -11,13 +11,13 @@ var ToolbarView = BrightcoveView.extend(
 		events : {
 			'click .view-list' :                   'toggleList',
 			'click .view-grid' :                   'toggleGrid',
-			'click .brightcove-toolbar':           'toggleToolbar',
+			'click .brightcove-tooltip':           'toggleTooltip',
```

Comment:
Why are you renaming this throughout?

Reply:
The button shows or hides the tooltip. I'm not sure why it was called toolbar, but it looked like it was misnamed to me. I kept reading through the code to figure out where it was toggling a toolbar or what that would mean, and finally for my own readability renamed it as I understood it.